cmd/infracost/run.go

```diff
@@ -352,20 +416,20 @@ func (r *parallelRunner) runProvider(job projectJob) (*projectOutput, error) {
 		return nil, clierror.NewCLIError(errors.New(m), "Cannot use Terraform state JSON with the infracost diff command")
 	}
 
-	m := fmt.Sprintf("Detected %s at %s", job.provider.DisplayType(), ui.DisplayPath(path))
-	if job.provider.Type() == "terraform_dir" {
-		m = fmt.Sprintf("Evaluating %s at %s", job.provider.DisplayType(), ui.DisplayPath(path))
-	}
+	name := job.provider.ProjectName()
+	displayName := ui.ProjectDisplayName(r.runCtx, name)
 
-	if r.runCtx.Config.IsLogging() {
-		log.Info().Msg(m)
-	} else {
-		fmt.Fprintln(os.Stderr, m)
-	}
+	logging.Logger.Info().Msgf("Starting evaluation for project %s", displayName)
```

Comment:
After running this on a few large repos I think we should move this and the 'Finished evaluation...' logs to debug since they create too many log lines breaking up any warnings. In the future we probably want to have some sort of progress indicator instead.